src/data/config.rs

```diff
@@ -84,7 +84,7 @@ pub struct ConfigFilePrebuilt {
 #[serde(deny_unknown_fields)]
 pub struct ConfigFileIndexes {
     pub index: String,
-    pub pub_key: Option<HashSet<String>>,
+    pub pub_key: HashSet<String>,
```

Comment:
In the config file pub_key can be excluded, so this needs to be optional.

Reply:
Done.